### DISCUSSION TITLE
Security Onion 2.3.40 VBoxLinuxAdditions.run fails with bzip2 error

### DISCUSSION THREAD

**Discussion by JDMcCADD** [Security-Onion-Solutions/securityonion#3842] [Unsupported Versions] [answered]
I've installed Security Onion 2.3.40 in VirtualBox 6.1.18 on a Mac OS 10.15.7 host.  so-status returns OK for all services.

I'm new to both Security Onion and Virtualbox and am unsure of my network config.  In VBox guest my SO mgmt interface is default NAT, and 2nd is internal with promiscuous enabled.  I tried to install VBox Guest Additions using...

sudo ./VBoxLinuxAddditions.run

Result is...

Verifying archive integrity... All good.
Uncompressing VirtualBox  6.1.18 Guest Additions for Linux........
bzip2 not found.  Please install: bzip2 tar; and try again.

I'm not a linux pro, but located instructions and attempted to install bzip2.  Seems I have no internet connectivity in the so-eval guest vm though, hence my effort to decipher my network config.

Are the VBox Guest Additioins for proper network functionality in SO 2.3.40 VM Guest?  If so, how do I install bzip2 so I can complete VBox Guest Additions install?

**Comment by TOoSmOotH** [comment, marked as answer]
sudo yum install bzip2

**Reply by JDMcCADD** [reply]
Progress - resolved my network issue, installed bzip2, and was able to run VBoxLinuxAdditions.run.  After several lines of output it failed with this error:

Virtualbox Guest Additions: Kernel headers not found for target kernel 3.10.0-1160.21.1.e17.x86_64

Wouldn't the kernel headers be included in the CentOS image in the Security Onion iso?
